toolchain/lowering/lowering_context.cpp

```diff
@@ -147,6 +147,8 @@ auto LoweringContext::BuildFunctionDefinition(SemIR::FunctionId function_id)
     for (const auto& node_id : semantics_ir().GetNodeBlock(block_id)) {
       auto node = semantics_ir().GetNode(node_id);
       CARBON_VLOG() << "Lowering " << node_id << ": " << node << "\n";
+      // clang warns on unhandled enum values; clang-tidy is incorrect here.
+      // NOLINTNEXTLINE(bugprone-switch-missing-default-case)
```

Comment:
What's the issue here? Is this something we could fix upstream in the `clang-tidy` checker? (No objection to the `NOLINT` in the mean time.)

Reply:
In case you missed this in the PR description:

> [bugprone-switch-missing-default-case](https://clang.llvm.org/extra/clang-tidy/checks/bugprone/switch-missing-default-case.html) is unhappy with EnumBase, but correctly identified yaml_test_helpers.cpp, so I'm opting to address it rather than disabling it even though it needs NOLINT in several locations as a result, in addition to what I think are some low-value default cases. I'd be fine going the other way with this too and disabling it globally (I could see it being noisier in the explorer).

I don't know what the root cause is; I'm not familiar enough with Clang AST to quickly investigate and fix it.